apps/ledger-live-mobile/src/newArch/features/FirmwareUpdate/components/UpdateBanner/ViewProps.tsx

```diff
@@ -0,0 +1,11 @@
+import { lastConnectedDeviceSelector } from "~/reducers/settings";
+
+export interface ViewProps {
+  bannerVisible: boolean;
+  lastConnectedDevice: ReturnType<typeof lastConnectedDeviceSelector>;
+  version: string;
+  onClickUpdate: () => void;
```

Comment:
```suggestion
  onClickUpdate(): void;
```
You can define a function type like this, more readable and simpler.